certified-connectors/inQuba Journey/apiDefinition.swagger.json

```diff
@@ -0,0 +1,309 @@
+                  "title": "Authorization Token"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Acquire Access Token",
+        "description": "Authenticate with the inQuba API and receive and authorization token that will be used for all API interactions.",
+        "operationId": "AcquireAccessToken",
+        "parameters": [
+          {
+            "name": "TenantName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tenant Name",
+            "description": "Name of Tenant as registered with inQuba."
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/x-www-form-urlencoded",
+            "description": "Request Content Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "HostURL",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Host of Token endpoint must be supplied without scheme name. Eg. api.inqubacx.com",
+            "x-ms-summary": "Host URL"
+          },
+          {
+            "name": "username",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Username as supplied by inQuba.",
+            "x-ms-summary": "Username"
+          },
+          {
+            "name": "password",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Password as supplied by inQuba.",
+            "x-ms-summary": "Password"
+          },
+          {
+            "name": "grant_type",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "default": "password",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "client_id",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client ID as supplied by inQuba.",
+            "x-ms-summary": "Client ID"
+          },
+          {
+            "name": "scope",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "default": "openid cx_api"
+          },
+          {
+            "name": "client_secret",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client Secret as supplied by inQuba.",
+            "x-ms-summary": "Client Secret"
+          }
+        ]
+      }
+    },
+    "/cems/api/Events": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+		     "type": "string",

```

Comment:
@prinayinq Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Thank you. Updated properties to match alignment as per the open source repo.